sound/soc/sof/intel/hda-mlink.c

```diff
@@ -228,6 +228,26 @@ static int hdaml_link_shutdown(u32 __iomem *lctl, int sublink)
 	return check_power_active(lctl, sublink, false);
 }
 
+static void hdaml_link_enable_interrupt(u32 __iomem *lctl, bool enable)
+{
+	u32 val;
+
+	val = readl(lctl);
+	if (enable)
+		val |= AZX_ML_LCTL_INTEN;
+	else
+		val &= ~AZX_ML_LCTL_INTEN;
+	writel(val, lctl);
```

Comment:
leave a blank line before as you will do in `hdaml_lctl_offload_enable()`

Reply:
done